### DISCUSSION TITLE
slCreateEngine failed to create an object

### DISCUSSION THREAD

**Discussion by yqazyzl** [android/ndk#1841] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20918327/219303694-b0f814cb-2fb3-4707-b7a4-ca26ea76d7ec.png)
Object creation failed, I want to know the reason for the failure to create, is there any log I can check?

Test equipment information：OPPO R9m | Android5.1

**Comment by DanAlbert** [comment, marked as answer]
https://developer.android.com/studio/debug/logcat
https://developer.android.com/studio/debug
https://developer.android.com/ndk/guides/debug

**Comment by EvansLuc** [comment]
@wlhq2015 , Did you find a resolution to this issue? I began to have the same issue after upgrading Android Studio from Dolphin to Electric Eel, including upgrading Gradle from 5.4.1 to 7.5, build tools from 30.0.0 to 33.0.0, and NDK from 21 to 25. There is no relevant logs in Logcat. This issue affects Android 5 and 6, but not 4.4, 10, 11, or 13.
